### ISSUE TITLE
[a11y] Small a11y issues with some icons

### ISSUE THREAD

**Issue by santialbo** [chakra-ui/chakra-ui#357]
The following icons are missing some a11y props
- `Buton.iconLeft` and `Button.iconRight` are missing `aria-hidden="true"`
- The icon in the alert is missing `aria-hidden="true"`
- The chevron on the `Select` element is missing `focusable="false"` and `aria-hidden="true"`

Is it better to submit a PR against master or against the TS branch?

**Comment by segunadebayo** [comment]
Thanks for this observation. It's better to PR against the TS branch

**Comment by segunadebayo** [comment]
Thanks for reporting this issue. I've just fixed this and updates will be released shortly

If after upgrade the problem still exists in your application please open a new issue and follow the instructions in the issue template.
